integration-tests/jail/run_compile.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "compile binaries"
+cd ./compile
+docker-compose up
```

Comment:
there's still this "network" for compiler -- how does this get shut down? running a compiler should be doable without creating a docker compose network

Reply:
i'll add docker teardown